tasks/extract_filepaths_for_granule/schemas/input.json

```diff
@@ -0,0 +1,244 @@
+                        "key",
+                        "bucket",
+                        "fileName",
+                        "path",
+                        "url_path",
+                        "type",
+                        "duplicate_found",
+                        "size"
+                      ]
+                    },
+                    {
+                      "type": "object",
+                      "properties": {
+                        "key": {
+                          "type": "string"
+                        },
+                        "bucket": {
+                          "type": "string"
+                        },
+                        "fileName": {
+                          "type": "string"
+                        },
+                        "path": {
+                          "type": "string"
+                        },
+                        "url_path": {
+                          "type": "string"
+                        },
+                        "type": {
+                          "type": "string"
+                        },
+                        "duplicate_found": {
+                          "type": "string"
+                        },
+                        "size": {
+                          "type": "integer"
+                        }
+                      },
+                      "required": [
+                        "key",
+                        "bucket",
+                        "fileName",
+                        "path",
+                        "url_path",
+                        "type",
+                        "duplicate_found",
+                        "size"
+                      ]
+                    }
+                  ]
+                }
+              },
+              "required": [
+                "granuleId",
+                "dataType",
+                "version",
+                "files"
+              ]
+            }
+          ]
+        }
+      },
+      "required": [
+        "granules"
+      ]
+    },
+    "config": {
+      "type": "object",
+      "properties": {
+        "glacier-bucket": {
+          "type": "string"
+        },
+        "protected-bucket": {
+          "type": "string"
+        },
+        "internal-bucket": {
+          "type": "string"
+        },
+        "private-bucket": {
+          "type": "string"
+        },
+        "public-bucket": {
+          "type": "string"
+        },
+        "file-buckets": {
+          "type": "array",
+          "items": [
+            {
+              "type": "object",
+              "properties": {
+                "regex": {
+                  "type": "string"
+                },
+                "sampleFileName": {
+                  "type": "string"
+                },
+                "bucket": {
+                  "type": "string"
+                }
+              },
+              "required": [
+                "regex",
+                "sampleFileName",
```

Comment:
I don't believe the `sampleFileName` field is required looking at the code.

Reply:
removed